PubHub/PubHub.API/Controllers/BooksController.cs

```diff
@@ -93,10 +93,10 @@ public async Task<IResult> GetBook(int id)
                 return Results.Problem(
                     statusCode: NotFoundSpecification.STATUS_CODE,
                     title: NotFoundSpecification.TITLE,
-                    detail: "We couldn't locate a book with the given ID",
+                    detail: "We couldn't locate a book with the given Id",
```

Comment:
I would prefer to use "ID" in the human friendly descriptions and `Id` in elsewhere.

Reply:
I do too. It might have been changed because I ran a Find&Replace in ID at some point :D